src/common/utils/customErrors/apiError.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import CustomError from "./customError";
+
+class APITokenError extends CustomError {
```

Comment:
@pichu07 Make this `APIError`. There is no error specifically required for tokens. Please make sure you use this error the correct way. You can have different error types in enums and use them.